metric/bigtable/src/main/java/com/spotify/heroic/analytics/bigtable/BigtableAnalyticsModule.java

```diff
@@ -87,13 +91,14 @@ public AnalyticsReporter reporter(final HeroicReporter reporter) {
     public Managed<BigtableConnection> connection(
         final AsyncFramework async, final ExecutorService executorService
     ) {
+        Optional<Integer> batchSize = empty();
```

Comment:
Sorry, I misunderstood you.
When you suggested 

> or declare as a default constant to follow the local convention?

 I thought you preferred that we assigned a default value to batchSize at Bigtable*Module creation time, and that is precisely what we would like to avoid so that Google's defaults get applied instead. I completely missed the inlining part. I'm adding a new commit for that now.

Also, good catch with the leftover `BIGTABLE_BULK_MAX_ROW_KEY_COUNT_DEFAULT` imports. That is precisely what I wanted to avoid :)